websites/B/Battlefield Portal/presence.ts

```diff
@@ -64,40 +64,29 @@ presence.on("UpdateData", async () => {
   }
 
   if (block) {
-    const selected = document.querySelector(".blocklySelected");
-    if (selected) {
-      if (
-        selected.querySelector(".subroutineBlockSubroutineText")
-          ?.textContent === "SUBROUTINE"
-      ) {
-        const blocklyText = selected.querySelectorAll(
-          "text[class=blocklyText]"
-        );
-        blocklyText.forEach(element => {
-          if (
-            element.parentElement.getAttribute("transform") ===
-            "translate(126.36666870117188,18)"
+    /* eslint-disable no-eval */
```

Comment:
I need to use `eval`, till ` getPageletiable`  can handle cyclic values

Reply:
use the one in the netflix presence